share/spice/just_delete_me/just_delete_me.js

```diff
@@ -0,0 +1,69 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_just_delete_me = function(api_result) {
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('just_delete_me');
+        }
+        var script = $('[src*="/js/spice/just_delete_me/"]')[0],
+            source = $(script).attr("src"),
+            query = source.match(/just_delete_me\/([^\/]+)/)[1],
+            decodedQuery = decodeURIComponent(query).split(" ")[0].toLowerCase();
+
+        if (!decodedQuery || decodedQuery.length < 4) {
+            return Spice.failed('just_delete_me');
+        }
+
+        var exact = api_result.filter(function(item) { return item.name.toLowerCase() === decodedQuery}),
+            exact = exact.length > 0 ? [exact[0]] : 
+                    api_result.filter(function(item) { return item.domains &&
+                        ((typeof(item.domains) === "string" && item.domains.toLowerCase() === decodedQuery) ||
+                         typeof(item.domains) === "object" && item.domains.some(function(domain) {return domain.toLowerCase() === decodedQuery}) )
+                    }),
+            api_result = exact.length > 0 ? [exact[0]] : api_result.filter(function (item) {
+                        item.domains =  !item.domains ? [] : typeof(item.domains) === "string" ? [item.domains] : item.domains;
+                        if (item.name.toLowerCase().includes(decodedQuery) || item.domains.some(function(domain) {return domain.toLowerCase().includes(decodedQuery);}))
+                            return true;
+                        });
+
+        if (!api_result || api_result.length < 1) {
+            return Spice.failed('just_delete_me');
+        }
+        Spice.add({
+            id: "just_delete_me",
+            name: "Answer",
+            meta: {
+                sourceName: 'Just Delete Me',
+                sourceUrl: "http://justdelete.me/#" + decodedQuery
+            },
+            data: api_result,
+            normalize: function(item) {
+                return {
+                    delete_url: item.url,
+                    title: item.name,
+                    url: "http://justdelete.me/#" + decodedQuery,
```

Comment:
I have to disagree here. As this is linked to from the name I would assume to see more details, especially as we will have the deletion URL listed separately, and clearly. Some pages also do not require any extra checks to delete your account, thus just clicking on the link will remove your account, if you are logged in. I think we should keep the item.name linking to more details on JustDeleteMe and keep the deletion URL separated.